docs/source/guide/webhooks.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Set up webhooks in Label Studio
+short: Webhooks Setup
+type: guide
+order: 650
+meta_title: Configure Webhooks in Label Studio
+meta_description: Label Studio documentation for setting up and configuring webhooks to integrate Label Studio with your machine learning pipeline
+---
+
+Webhooks in Label Studio let you set up integrations that subscribe to certain events that occur inside Label Studio. When an event is triggered, Label Studio sends an HTTP POST request to the configured webhook URL. For example:
+```json
+{
+    "action": "TASK_CREATED",
+    "tasks": [
+        {"id": 1, ...},
+        ...
+    ]
+}
+```
+
+Your application or service can then respond to that event information however you want. 
+
+## What to use Label Studio webhooks for 
+
+Some events that happen in Label Studio happen in response to an API call. For those, the API response is sufficient to know that the action has occurred. But other events happen inside the Label Studio UI, such as project changes and task annotation activities. To find out immediately when those changes occur, you can use webhooks to notify your application or machine learning pipeline to take specific actions. 
+
+The actions your webhook takes in your application or pipeline can vary depending on the event. For example, you might use events sent to your webhook URL to do the following:
+- Start training a machine learning model after a certain number of tasks have been annotated.
+- Perform active learning after a task has been annotated.
+- Prompt annotators to begin working on a project after it is fully set up.
+- Create a new version of training data in a dataset versioning repository. 
+
+Limit the number of requests to your server by subscribing only to the events relevant for your use case.
+
+## Available Label Studio webhooks
+Label Studio makes two main types of events available to integrate with webhooks: project-level task events and organization events.
+
+<table>
+  <tr>
+    <th>Event Action</th>
+    <th>Details</th>
+    <th>Use Case</th>
+  </tr>
+  <tr>
+    <td>[Task Created](webhook_reference.html#Task-Created)</td>
+    <td>For a specific project, triggers when new tasks are created. One event per task.</td>
+    <td>Use to take action in your machine learning pipeline. </td>
+  </tr>
+  <tr>
+    <td>[Task Deleted](webhook_reference.html#Task-Deleted)</td>
+    <td>For a specific project, triggers when tasks are deleted. One event per deleted task.</td>
```

Comment:
again, 1 event per batch removal @chiganov is it right?

Reply:
Task deleted and Annotation deleted - `many=true`

Project deleted - `many=False`

Reply:
Not, it is not right:
```
{
    "action": "TASKS_DELETED",
    "tasks": [
        {
            "id": 18
        },
        {
            "id": 19
        },
        {
            "id": 20
        },
        {
            "id": 21
        },
        {
            "id": 22
        },
        {
            "id": 23
        }
    ],
...
...
...
```

Reply:
thanks!